_config.yml

```diff
@@ -44,10 +44,13 @@ exclude:
   - .sass-cache/
   - .jekyll-cache/
   - assets/scss
+  - bin
```

Comment:
This isn't currently used in this repo, but bundle binstubs might still be created locally depending on your dev env and should be ignored